File: ipc/src/data/get_by_name.ts
(async () => {
  let resp = await fetch("http://localhost:8080", {
    method: "POST",
    body: JSON.stringify({
      jsonrpc: "2.0",
      id: 1,
      method: "data/get",
      params: {
        "@type": "FlowProperty",
        "name": "Mass",
      }
    })
  });
  let v = await resp.json();
  console.log(v);
  // {
  //   jsonrpc: "2.0",
  //   result: {
  //     "@type": "FlowProperty",
  //     "@id": "93a60a56-a3c8-11da-a746-0800200b9a66",
  //     name: "Mass",
  //     category: "Technical flow properties",
  //     version: "00.00.000",
  //     flowPropertyType: "PHYSICAL_QUANTITY",
})();

File: docs/ipc/data/get_by_name.py
import olca_ipc as ipc
import olca_schema as o

client = ipc.Client()
mass = client.get(o.FlowProperty, name="Mass")
print(mass)
# FlowProperty(
#   name='Mass',
#   id='93a60a56-a3c8-11da-a746-0800200b9a66',
#   unit_group=Ref(id='93a60a57-a4c8-11da-a746-0800200c9a66', ...
